editor/scene_tree_dock.cpp

```diff
@@ -3268,6 +3307,27 @@ void SceneTreeDock::_script_dropped(const String &p_file, NodePath p_to) {
 		undo_redo->add_undo_method(ed, "live_debug_remove_node", NodePath(String(edited_scene->get_path_to(n)).path_join(new_node->get_name())));
 		undo_redo->commit_action();
 	} else {
+		// Check if dropped script is compatible.
+		if (n->has_meta(SceneStringName(__custom_type_script))) {
+			Ref<Script> ct_scr = n->get_meta(SceneStringName(__custom_type_script));
+			if (!scr->inherits_script(ct_scr)) {
+				StringName custom_type_name = ct_scr->get_global_name();
```

Comment:
```suggestion
				String custom_type_name = ct_scr->get_global_name();
```
This is purely for displaying in a warning, so it can be string. Especially when the 2 strings below are also String (when you remove SNAME).